### PR TITLE
tests: add tests for smb.version keyword v3

### PR DESCRIPTION
**Pull request by jmtaylor90**
## Ticket

If your pull request is related to a Suricata ticket, please provide
the full URL to the ticket here so this pull request can monitor
changes to the ticket status:

Redmine ticket:
https://redmine.openinfosecfoundation.org/issues/5075

### CODE REVIEW

**Comment by jmtaylor90** [comment]
- Added pcap origination information
- Included original author commits

**Comment by jufajardini** [comment]
Current Suricata PR: https://github.com/OISF/suricata/pull/9451

This PR per se looks good to me, but as it seems that the Suri PR's checks have not picked this PR, I'll wait for the next PR version there to check for the CI results, then come back to this :)

File: tests/smb-version-keyword-invalid/test.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+requires:
+  min-version: 7
```
Comment (catenacyber) [inline tests/smb-version-keyword-invalid/test.yaml:2]: Please make it 8

Comment (jmtaylor90) [reply tests/smb-version-keyword-invalid/test.yaml:2]: Is it okay to force push that update or should I make a new PR?

Comment (catenacyber) [reply tests/smb-version-keyword-invalid/test.yaml:2]: For me it is ok, but maybe better to do a new PR ;-)

Comment (jmtaylor90) [reply tests/smb-version-keyword-invalid/test.yaml]: updated

**Comment by catenacyber** [comment]
@jmtaylor90 did you just rebase and force-push ? for https://github.com/OISF/suricata/pull/10027 CI to be green ?
(or were there other changes ?)

**Comment by jmtaylor90** [comment]
> @jmtaylor90 did you just rebase and force-push ? for https://github.com/OISF/suricata/pull/10027 CI to be green ?
> (or were there other changes ?)
> 

Yeah, was just a rebase and force push, no other changes.

**Review by victorjulien — APPROVED** [review]

**Comment by victorjulien** [comment]
Merged in #1593, thanks!

**Pull request closed without merging**